GrievanceManagement/src/main/java/com/grievance/dto/CommentsDto.java

```diff
@@ -0,0 +1,124 @@
+package com.grievance.dto;
+
+import com.grievance.entity.Ticket;
+import java.util.Date;
+
+/**
+ * Comments In DTO.
+ */
```

Comment:
CommentsDto can be renamed to CommentsInDto, keep in and out dtos distinguishable.